src/modules/Builder/components/Configuration/components/Network/Network.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import * as RPC from "api/rpc/blackmagic";
+
+import Select from "common/components/Select";
+
+import Card from "common/containers/Card";
+import Input from "common/components/Input";
+
+import { timeZones } from "./config";
+
+export default class Network extends Component {
+    static formatTimeZone() {
+        return timeZones.map((item) => ({
+            value: item,
+            text: item,
+        }));
+    }
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            hostName: "",
+            timeZone: timeZones[0],
+            warningStatus: false,
+        };
+
+        this.onInputChange = this.onInputChange.bind(this);
+        this.onTimeZoneChange = this.onTimeZoneChange.bind(this);
+        this.hostNameValidor = this.hostNameValidor.bind(this);
+    }
+
+    componentDidMount() {
+        RPC.fetchDefaultConfig()
+            .then((defaultConfig) => {
+                this.setState(() => ({ hostName: defaultConfig.hostname }));
+            });
+    }
+
+    onInputChange(name, value) {
+        this.setState(() => ({
+            [name]: value,
+        }));
+    }
+
+    onTimeZoneChange(value) {
+        const timeZone = timeZones.find((item) => item === value);
+        this.setState(() => ({
+            timeZone,
+        }));
+    }
+
+    hostNameValidor(value) {
```

Comment:
You need to make sure that all validation functions have one setState call and one return.